changelogs/fragments/67205-pvlan-config-vmware-dvs-portgroup.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+- vmware_dvs_portgroup - Added support for dpg with private VLAN.
```

Comment:
`dpg` is a bit confusing. Could you write `distributed port group` instead?

Reply:
Of course, it's changed now.